usr/share/lib/img_proof/tests/conftest.py

```diff
@@ -146,31 +147,47 @@ def f():
 
 
 @pytest.fixture()
-def is_suma_server(host, get_baseproduct):
+def is_suma_server(host, get_baseproduct, get_suma_version):
     def f():
         suma_server_product = '/etc/products.d/SUSE-Manager-Server.prod'
         suma_server = host.file(suma_server_product)
         base_product = get_baseproduct()
-
+        suma_version = get_suma_version(suma_server_product)
+
+        if suma_version and suma_version.startswith('4'):
+            # For suma 4.3 baseproduct HAS to be SUMA proxy
```

Comment:
Although this is just a comment, is it true that in case of SUMa Server the baseproduct has to be SUMa proxy?

Reply:
Nope. It  was a copy paste issue. I'll fix it right away